src/middlewares/emptybody.js

```diff
@@ -0,0 +1,20 @@
+import responseMessage from '../helpers/responseMessage';
```

Comment:
Good job. Please rename to `emptyBodyValidator` since that's what it does. the current name doesn't clearly tell me it's function.